BC/FilteredNoiseInflowGenerator/readMe.md

```diff
@@ -0,0 +1,41 @@
+# Update by Lukas Lebovitz (08/2017)
+
+- This variant of the filtered noise inflow generator uses the proposed method of selective filtering. Selective filtering only filters the virtual grid coordinates which are nearest to actual mesh faces. This can heavily improve performance for grids that are non-uniform (i.e. meshes with refinements). 
+
+- This version fixes a bug that crashes the simulation sometimes when the number of indices on the virtual grid can't be split equally to all processors.
+
+- The `getRandomField()` function now works in parallel and distributes random number generation on all available processors.
+
+- This version fixes a bug that crashes the simulation sometimes when the number of indices on the virtual grid can't be split equally to all processors.
+
+- Furthermore the indices are now distributed to the processors according to their computational load which is depending on the corresponding integral length scales. This only works if the integral length scales are vertically increasing (in z-direction).
```

Comment:
What happens if they are not increasing, does the code crash?

Reply:
No. However, computational load will not be distributed homogeneously on processors.